### ISSUE TITLE
 fzf-tmux bugs after recent commits

### ISSUE THREAD

**Issue by atweiden** [junegunn/fzf#442]
A very recent commit seems to cause an error with `**` tab completion:

```
vim file**
bash: fzf-tmux -d 40%: command not found
```

It also breaks fzf.vim features when a pane is zoomed in a multi-pane tmux window (e.g. `:Files`). The features don't break in unzoomed panes in multi-pane windows, or in single-pane windows.

Tested on Arch with rebuilt 0.11.1 release.

**Comment by junegunn** [comment]
Hi, thanks for the report. Yes, we did update the script so that fzf-tmux can still split the screen even if the pane is zoomed (see #433), but I can't seem to reproduce the problem you described. Can you check what's going on with `#!/bin/bash -x`?

/cc @pokey

**Comment by atweiden** [comment]
After adding `-x` to the fzf-tmux executable, I'm not able to get a debug message. When the error occurs, I can either hit <kbd>Enter</kbd> to do `vim file**`, <kbd>Ctrl-C</kbd> to cancel, or <kbd>Tab</kbd> a few more times to get a regular readline tab completion.

The command `fzf-tmux -d 40%` works fine on the console:

```
$ fzf-tmux -d 40%
+ args=()
+ opt=
+ skip=
+ swap=
+ close=
+ term=
+ '[' 2 -gt 0 ']'
+ arg=-d
+ case "$arg" in
+ '[' -n '' ']'
+ [[ -d =~ ^.[lrw] ]]
+ opt=
+ [[ -d =~ ^.u ]]
+ '[' 2 -gt 2 ']'
+ shift
+ [[ 40% =~ ^[0-9]+%?$ ]]
+ size=40%
+ [[ 40% =~ %$ ]]
+ size=40
+ '[' -n '' ']'
+ opt=' -p 40'
+ shift
+ '[' 0 -gt 0 ']'
+ '[' -z %27 ']'
+ tmux list-panes -F '#F'
+ grep -q Z
+ fzf
+ exit 130
```

**Comment by atweiden** [comment]
I had forgotten to diff my fzf-tmux executable (the one included in the 0.11.1 release), with git HEAD. For Arch users, this is resolved with [fzf-git](https://aur.archlinux.org/packages/fzf-git/) until the next point release.

**Comment by junegunn** [comment]
A fresh clone of this repo gives fzf-tmux with correct execute permission.
